test/run

```diff
@@ -26,7 +26,7 @@ case $TEST_SCENARIO in
             exit 1
         fi
         test/image-prepare --quick --containers --verbose $TEST_OS
-        test/common/run-tests --test-dir test/containers --test-glob check-"$container"'*'
+        test/common/run-tests--test-dir test/containers --test-glob check-"$container"'*' --track-naughties
```

Comment:
You dropped the first space accidentally, that's not going to work.

Reply:
Ouch, hacking too fast...